mudb/src/server.ts

```diff
@@ -40,8 +40,8 @@ export class MuServerProtocol<Schema extends MuAnyProtocolSchema> {
     public readonly server:MuServer;
     public readonly clients:{ [sessionId:string]:MuRemoteClientProtocol<Schema['client']> } = {};
 
-    public broadcast:MuMessageInterface<Schema['client']>['userAPI'];
-    public broadcastRaw:(bytes:Uint8Array|string, unreliable?:boolean) => void;
+    public broadcast!:MuMessageInterface<Schema['client']>['userAPI'];
```

Comment:
Maybe make these variables `|undefined`

Reply:
couldn't just do that. it'll break other modules. i'll try to see if i could move the definitions into the constructor

Reply:
gonna break other modules. maybe also give these a default value instead?